### DISCUSSION TITLE
About Total Latency

### DISCUSSION THREAD

**Discussion by thisleo** [alvr-org/ALVR#1030] [Q&A] [answered]
![aGxUQw1649409893374](https://img.dopobo.com/uPic/aGxUQw1649409893374.png)
What does this idle represent, and can it be reduced?

**Comment by zmerp** [comment, marked as answer]
The idle latency is the time between when a frame is ready and when it is actually shown on screen. To reduce this waiting time, the server and client rendering cycles need to be synchronized. This could trivially be achieved when switching to OpenXR, which supports Phase Sync.
